src/flamegraph/mod.rs

```diff
@@ -305,3 +324,18 @@ fn deannotate(f: &str) -> &str {
     }
     f
 }
+
+impl Default for Direction {
```

Comment:
Do we even need an `impl Default for Direction`? It doesn't seem to be used anywhere?

Reply:
I think we need Direction to implement Default since Options does.

Reply:
Given that we [directly construct `Options`](https://github.com/jonhoo/inferno/pull/48/files#diff-5b9f024b0146520495e6e19fa806f9daR47), why does `Options` have to `impl Default` then? :p

Reply:
What about when someone wants to use it as a library? Wouldn't we want the ability to just use the default options?

Reply:
Ah, hmm, I suppose that's true... In fact, arguably what we should do in `from_args` is construct a `default` and then only override with what the args say...